### PR TITLE
Update azure_rm_virtualmachinescalesetextension related document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_virtualmachinescalesetextension related document.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_virtualmachinescalesetextension/azure_rm_virtualmachinescalesetextensioin_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescalesetextension_facts.py [hunk 1]
```diff
@@ -57,62 +57,63 @@
...
             description:
-                - Virtual machine name
+                - Virtual machine name.
             returned: always
             type: str
             sample: myvmss
         name:
             description:
-                - Virtual machine name
+                - Virtual machine name.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescalesetextension_facts.py]: Are `vmss_name` and `name` different? I'm guessing they are: 
```suggestion
                - Virtual machine extension name.
```

File: lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescalesetextension.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 short_description: Managed Azure Virtual Machine Scale Set extension
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescalesetextension.py]:
```suggestion
short_description: Manage Azure Virtual Machine Scale Set (VMSS) extensions
```

File: lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescalesetextension.py [hunk 3]
```diff
@@ -22,7 +22,7 @@
...
 
 description:
-    - Create, update and delete Azure Virtual Machine Extension
+    - Create, update and delete Azure Virtual Machine Extension.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescalesetextension.py]:
```suggestion
    - Create, update and delete Azure Virtual Machine Scale Set (VMSS) extensions.
```

File: lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescalesetextension_facts.py [hunk 4]
```diff
@@ -57,62 +57,63 @@
...
             description:
-                - Resource group name
+                - Resource group name.
             returned: always
             type: str
             sample: myrg
         vmss_name:
             description:
-                - Virtual machine name
+                - Virtual machine name.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescalesetextension_facts.py]:
```suggestion
                - Virtual machine scale set name.
```

**Review by acozine — COMMENTED** [review]
Thanks @Fred-sun. Couple of comments on this one.

File: lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescalesetextension_facts.py [hunk 5]
```diff
@@ -57,62 +57,63 @@
...
             description:
-                - Virtual machine name
+                - Virtual machine scale set name.
             returned: always
             type: str
             sample: myvmss
         name:
             description:
-                - Virtual machine name
+                - Virtual machine extention name.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescalesetextension_facts.py]:
```suggestion
                - Virtual machine extension name.
```

**Comment by Fred-sun** [comment]
ready_for_review

**Comment by acozine** [comment]
Looks good, thanks @Fred-sun.

**Pull request merged**